### PR TITLE
Include avatar placeholder in relevant places

### PR DESCRIPTION
**Pull request by PVince81**
## Description
Pass in the user display name for the fallback logic that displays
letter.
Quick fix for alignment of avatar in top bar.
Update ODS for the fix for setting avatar size.
Use avatar placeholder component also in collaborator autocomplete and list.

## Related Issue
None raised

## Motivation and Context
Top bar looking shitty without proper alignment.
Avatar placeholder had missing letters.

## How Has This Been Tested?
- [x] TEST: with placeholder: letters are shown, alignment is ok
- [x] TEST: with avatar picture: picture is shown in circle, alignment is ok

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by PVince81** [comment]
- [ ] investigate why button height is reduced in responsive mode

**Comment by PVince81** [comment]
fixed test errors because the avatar placeholder had included itself in the places where the test code is trying to read the user name

File: apps/files/src/components/Collaborators/AutocompleteItem.vue [hunk 1]
```diff
@@ -2,13 +2,13 @@
   <div class="uk-flex uk-flex-middle">
     <oc-spinner v-if="loading" uk-spinner="ratio:1.6" class="uk-margin-small-right" :aria-label="$gettext('Loading avatar')" />
     <template v-else>
-      <oc-avatar v-if="avatar" :src="avatar" class="uk-margin-small-right" width=50 height=50 />
+      <oc-avatar v-if="item.value.shareType === 0" :src="avatar" class="uk-margin-small-right" width=50 height=50 :userName="item.label" />
```
Comment (kulmann) [inline apps/files/src/components/Collaborators/AutocompleteItem.vue]: still has the obsolete `height` param

File: apps/files/src/components/Collaborators/Collaborator.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <div class="files-collaborators-collaborator-information uk-flex uk-flex-wrap uk-flex-middle">
       <oc-spinner v-if="loading" key="collaborator-avatar-spinner" uk-spinner="ratio:1.6" class="uk-margin-small-right" :aria-label="$gettext('Loading')"/>
       <div v-else key="collaborator-avatar-loaded">
-        <oc-avatar v-if="avatar" :src="avatar" class="uk-margin-small-right" width=50 height=50 />
+        <oc-avatar v-if="collaborator.info.share_type === '0'" :src="avatar" class="uk-margin-small-right" size=50 :userName="collaborator.displayName" />
```
Comment (kulmann) [inline apps/files/src/components/Collaborators/Collaborator.vue]: This should still be `width` instead of `size`, right?!

Comment (PVince81) [reply apps/files/src/components/Collaborators/Collaborator.vue]: indeed :-/

**Comment by PVince81** [comment]
@kulmann fixed.

Also I've fixed CI because now the avatar placeholder text was sneaking into the string that is read for assertions.

**Pull request merged**